.pre-commit-config.yaml

```diff
@@ -17,7 +17,12 @@ repos:
     rev: v2.4.1
     hooks:
       - id: codespell
-        args: ["doc examples examples_trame pyvista tests", "*.py *.rst *.md"]
+        args:
+          [
+            "-L nd",
```

Comment:
Can we move this to this file?
https://github.com/pyvista/pyvista/blob/e79eaeafe7be173472c3c1b3f39111af78b319c9/pyproject.toml#L190